[WIP] features/directives/checklist

bisht-richa opened the pull request:
- Created new directives for check list.
- Created separate css for check list.

Commit:
Create checklist directive

Commit:
Create checklist directive

Commit:
Add Unit Tests for Admin Repo checks controller

Commit:
Merge branch 'master' of github.com:crowbar/crowbar-angular into features/upgrade7/repository-checks-page/repository-checks-UI

santiph commented on assets/app/widgets/crowbar-checklist/crowbar-checklist.directive.jade:
```diff
@@ -0,0 +1,14 @@
+            'glyphicon-ok-circle': check.status && completed,\
+            'glyphicon-remove-circle': !check.status && completed\
+        }")
+        span(translate='{{::check.label}}') 
```
Missing new line at the end of the file

Commit:
Revert back directive implementation changes from landing page

Commit:
Revert back directive implementation changes from landing page

santiph commented:
Looks good to me. Approved

santiph committed:
Merge pull request #20 from crowbar/features/upgrade7/repository-checks-page/repository-checks-UI

Unit Tests for Admin Repo Checks

Commit:
merge master

Commit:
Fix styles and add directive namespace to LESS styles

Commit:
Fix styles and add directive namespace to LESS styles

Commit:
Add new line at the end of the file

Commit:
Add new line at the end of the file

skazi0 committed:
Admin Server Upgrade page base

skazi0 committed:
- added dummy backend for admin upgrade
- added factory/service to wrap the api on the ui side
- added api calls for starting and periodic checking of upgrade status
- added part of tests

skazi0 committed:
added more tests

skazi0 committed:
added checkAdminUpgrade() tests

skazi0 committed:
post review changes

skazi0 committed:
increased severity of curly braces check

santiph committed:
Merge pull request #25 from skazi0/eslint-curly

increased severity of curly braces check

santiph committed:
Merge pull request #23 from skazi0/features/upgrade7/upgrade-admin-server/page-ui

Features/upgrade7/upgrade admin/upgrade admin ui

skazi0 committed:
switched from disabled class to ng-disabled

santiph committed:
Merge pull request #27 from skazi0/disabled-fix

switched from disabled class to ng-disabled

Commit:
Merge branch 'features/directives/checklist' of github.com:crowbar/crowbar-angular into features/directives/checklist

Commit:
Revert "Merge branch 'features/directives/checklist' of github.com:crowbar/crowbar-angular into features/directives/checklist"

This reverts commit ca2462370231cce8036ad6618f36c41043d39e43, reversing
changes made to db82de2b4b840e73379c89643e160a34fed6c30c.

Commit:
Revert "Add new line at the end of the file"

This reverts commit a096941fe1c544160b17cf2deaf2e99ad5789f9a.

Commit:
Revert "Revert "Add new line at the end of the file""

This reverts commit 383ff5af3786dba08e22de57c581a1ab09af3bd1.

Commit:
Revert "Revert "Merge branch 'features/directives/checklist' of github.com:crowbar/crowbar-angular into features/directives/checklist""

This reverts commit b0f7a6430ca5082e9ad15c2cba15f919afe78dd7.

skazi0 commented:
Replaced by https://github.com/crowbar/crowbar-ui/pull/28
